components/tools/OmeroJava/test/integration/MetadataServiceTest.java

```diff
@@ -844,7 +844,7 @@ public void testLoadAnnotationsUsedNotOwned() throws Exception {
         List<IObject> result = f
                 .getMetadataService()
                 .loadAnnotationsUsedNotOwned(TagAnnotation.class.getName(), id1);
-        Assert.assertTrue(result.size() > 0);
+        Assert.assertNotEquals(result.size(), 0);
```

Comment:
This kind of thing might be more obvious as,
```java
Assert.assertFalse(result.isEmpty());
```
...? Perhaps just a matter of taste.